st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -73,6 +74,7 @@
 ST2_LOG_FILES_PATH = '/var/log/st2/*.log'
 MISTRAL_LOG_FILES_PATH = '/var/log/mistral*.log'
 
+RPM_OUTPUT_FILE_PATH = "/tmp/installed_rpm_info.txt"
```

Comment:
I don't think stackstorm uses RPMs.  This is specific to our product.  Is this path something that can made generic and the path be loaded from a config file?